SpatialGDK/Documentation/welcome.md

```diff
@@ -0,0 +1,18 @@
+# Welcome to the SpatialOS GDK for Unreal 
+
+<img src="{{assetRoot}}assets/unrealgdk-headline-image.png" style=" margin: 0 auto; display: block; width: 60%" />
+
+<br/>
+
+<%(TOC)%>
+
+The SpatialOS Game Development Kit (GDK) for Unreal is a <a href="" data-track-link="Plugin Link Clicked|product=Docs|platform=Win|label=Win" target="_blank">plugin (Unreal documentation)</a>  which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API. 
+
+The GDK offers:
+* **Multi-server support:** leveraging our cloud platform [SpatialOS (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/concepts/spatialos), the GDK turns Unreal’s single-authority engine into a distributed one,  enabling you to use multiple Unreal servers to make games with more players, Actors, and gameplay systems than previously possible.
+* **An Unreal-native experience:** keeping traditional workflows and networking APIs that Unreal Engine developers are familiar with, the GDK introduces new native-feeling concepts that turn a single-authority engine into a distributed one. This enables the GDK to retain the functionality of the networking features which Unreal offers out of the box, including transform synchronization, character movement controller, client-side prediction.
+* **An easy way to get started:** we have made sure it’s easy to get started with the GDK by including a Starter Project which you can use as a tour of SpatialOS and a base for your own game, as well as a guide to porting your current multiplayer Unreal game to run on SpatialOS.
+
+#### [Get started]({{urlRoot}}/content/get-started/introduction.md) with the SpatialOS GDK for Unreal.
+
+<%(Callout type="warn" message="This is an [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal, pending stability and performance improvements. The API may change as we learn from feedback  - see the guidance on [Recommended use]({{URLRoot}}/content/recommended-use).")%>
```

Comment:
Do we still need this callout? Thought we nuked them.

Reply:
Yep I think this is the main place where we do such a callout

Reply:
Yes